caddyconfig/httpcaddyfile/directives.go

```diff
@@ -63,6 +63,7 @@ var directiveOrder = []string{
 	"push",
 
 	// handlers that typically respond to requests
+	"error",
```

Comment:
Is this at the optimal place? (Sometimes I wish I had reversed the positions of `file_server` and `reverse_proxy`.)

For example, will it be more common to use this directive as a "fallback" (i.e. no matchers) or as a way to handle a certain class of requests (i.e. with matchers)? If a fallback, should be at the end. If specific, it's probably in a reasonable place. (As to the exact placement, though, it's still an open question.)

Reply:
Honestly, I'm not sure, but I'm leaning toward people using matchers with `error` more often than not :man_shrugging:

Reply:
Hmm yeah I'm not sure either. I feel like returning an error could be a common fallback scenario ("Oh, if we haven't handled it yet, then return an error instead of the default 200 OK"). Then again, I can also see how the opposite could be common: "If we haven't errored, then reverse_proxy / file_server." I kinda think the first may be more common. Should we move it to the end?

Reply:
Sure why not :man_shrugging: